addons/fleet/views/fleet_vehicle_views.xml

```diff
@@ -117,7 +117,7 @@
                             <group>
                                 <group col="1" name="fiscality_group" string="Fiscality">
                                     <group col="1" name="fiscality_first_group">
-                                        <field name="horsepower_tax" widget="monetary"/>
+                                        <field name="horsepower_tax" widget="monetary" invisible="power_unit != 'horsepower'"/>
```

Comment:
Fiscal horsepower is not related to the unit of measure but to the power of the car in general so it should always be displayed